ova/provision.sh

```diff
@@ -25,13 +28,20 @@ echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
 # Dependences
 yum install openssl -y
 
-install_wazuh
 
-elastic_stack_${ELK_MAJOR}
+healthCheck
```

Comment:
This health check fails when using 4GB of memory, we need to remove it.

Reply:
Done in 71fd88cf61ad859245d50e315178f0058cb61754